src/ontology/Makefile

```diff
@@ -1002,3 +1002,21 @@ bg-clean:
 
 succeed:
 	echo make succeeded
+	
+
+###### NICO MIGRATION: overwriting unsuccessful goals!
+
+../sparql/construct-ecs-from-xrefs.sparql:
```

Comment:
this is unnecessary - you only needed to do this as you didn't have the sparql folder mounted in Docker

Reply:
See email, not sure thats the real problem here.